src/operator-manual/metrics/02-reference.md

```diff
@@ -0,0 +1,31 @@
+# Metrics Reference
+
+Kubewarden exposes some relevant metrics that enhaces visibility of the platform, and allows cluster
+administrators and policy developers to identify patterns and potential issues.
+
+## Policy Server
+
+The Policy Server is the component that initializes and runs policies. Upon receiving requests from
+the Kubernetes API server, it will forward the request to the policy, and return the response
+provided by the policy to the Kubernetes API server.
+
+### Metrics
+
+Name | Type | |
+--- | --- | ---
+`kubewarden_policy_evaluations_total` | Counter | [Baggage](#kubewarden_policy_evaluations_total)
+
+#### `kubewarden_policy_evaluations_total`
+
+##### Baggage
+
+Label | Description
+--- | ---
+`policy_name` | Name of the policy
+`resource_name` | Name of the evaluated resource
+`resource_kind` | Kind of the evaluated resource
+`resource_namespace` | Namespace of the evaluated resource. Not provided if the resource is cluster scoped.
```

Comment:
```suggestion
`resource_namespace` | Namespace of the evaluated resource. Empty if the resource is cluster scoped.
```

Reply:
Is not empty, is not present in the baggage at all

Reply:
Example: https://github.com/kubewarden/policy-server/blob/81279c84fa42323dc864029da52eb8827319388c/src/metrics/policy_evaluations_total.rs#L36-L41

Reply:
Updated to "Not present"